public/app/percona/inventory/Tabs/Nodes.tsx

```diff
@@ -51,6 +51,11 @@ export const NodesTab = () => {
   const styles = useStyles2(getStyles);
   const dispatch = useAppDispatch();
 
+  const mappedNodes: Node[] = useMemo(
```

Comment:
Same as in `Agents.tsx`, no need to specify the resulting type `Node[]` two times.